Lib/pydoc.py

```diff
@@ -52,7 +52,8 @@ class or function within a module or module in a package.  If the
 #   - If the __file__ attribute on a module is a relative path and
 #     the current directory is changed with os.chdir(), an incorrect
 #     path will be displayed.
-
+import ast
```

Comment:
Don’t delete the empty line that was added for breathing

```suggestion

import ast
```